### PR TITLE
build(deps): bump @stoplight/types from 9.1.2 to 11.0.0

### PR DESCRIPTION
**Pull request by lag-of-death**
## Changes
This PR updates its dependencies to work with adjusted `IHttpOperation` type from `@stoplight/types` (https://github.com/stoplightio/types/pull/50) because of https://github.com/stoplightio/prism/pull/514.

(Temporarily, `dist` folder is included.)

## Dependencies 
https://github.com/stoplightio/json-ref-resolver/blob/816018a55279136fba340b47ba19ce5ef550e737/package.json#L48

## Plan:
- merge `@stoplight/types`
- update dependency on `@stoplight/types`
- have an approval
- remove `dist` folder
- merge

This PR should be merged second, after `@stoplight/types`.

**Why github branches and no `yalc`? I was trying `yalc` first, but were having issues with nested `.yalc` folders (like `.yalc/@stoplight/something/.yalc/@stoplight/something_else`). Yarn, for some reason, couldn't have resolved dependencies.**

## Order of merging:
1. `@stoplight/types` (https://github.com/stoplightio/types/pull/50)
2. `@stoplight/json-ref-resolver` (https://github.com/stoplightio/json-ref-resolver/pull/86)
3. `@stoplight/http-spec` (https://github.com/stoplightio/http-spec/pull/15) and `@stoplight/spectral` (https://github.com/stoplightio/spectral/pull/444)
4. `@stoplight/graphite` (https://github.com/stoplightio/graphite/pull/211) and `@stoplight/prism` (https://github.com/stoplightio/prism/pull/514)

### CODE REVIEW

**Comment by pytlesk4** [comment]
@lag-of-death why is the dist folder included?

**Comment by lag-of-death** [comment]
@pytlesk4, yalcing `types` was the idea. But I was having issues witch deeper dependencies like `.yalc/@stoplight/something/.yalc/@stoplight/types`

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "@stoplight/json": "^2.2.2",
-    "@stoplight/types": "^9.1.2",
+    "@stoplight/types": "git://github.com/stoplightio/types.git#fix/do-not-flatten-security",
```
Comment (P0lip) [inline package.json]: Let's use npm version of types.

Comment (lag-of-death) [reply package.json]: @P0lip, we don't have npm version of types with the changes in it (`IHttpOperation`), or do we?

Comment (P0lip) [reply package.json]: Let's release it first and then bump the version.

**Comment by lag-of-death** [comment]
@P0lip, are you OK with the changes?

**Comment by pytlesk4** [comment]
@lag-of-death delete the branch please, you leave them all the time.

**Comment by lag-of-death** [comment]
@pytlesk4, I beat my breast with humility

**Comment by pytlesk4** [comment]
All good @lag-of-death, I got you.

**Pull request merged**